Content.Server/_Harmony/Uncloneable/UncloneableSystem.cs

```diff
@@ -0,0 +1,19 @@
+using Content.Shared.Cloning.Events;
+using Content.Server._Harmony.Uncloneable;
+
+namespace Content.Server._Harmony.Uncloneable;
+
+public sealed class UncloneableSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+       SubscribeLocalEvent<UncloneableComponent, CloningAttemptEvent>(OnCloningAttempt);
+    }
+
+    private void OnCloningAttempt(Entity<UncloneableComponent> ent, ref CloningAttemptEvent args)
+    {
+            args.Cancelled = true;
```

Comment:
Indentation has 4 extra spaces
```suggestion
        args.Cancelled = true;
```